check.ts

```diff
@@ -0,0 +1,140 @@
+import { Request, Response } from 'express';
+
+import ReviewService from '@/services/reviewService';
+import type { PullRequestWebhookEvent, PullRequestReviewWebhookEvent, IssueCommentWebhookEvent } from '@/types';
+import config from '@/utils/config';
+import logger from '@/utils/logger';
+
+class WebhookController {
+  private reviewService: ReviewService;
+
+  constructor() {
+    this.reviewService = new ReviewService();
+  }
+
```

Comment:
Missing webhook signature verification for security. GitHub webhooks should be validated using the X-Hub-Signature-256 header to ensure requests are authentic. This is a critical security vulnerability.

```suggestion
async handlePullRequest(req: Request<{}, {}, PullRequestWebhookEvent>, res: Response): Promise<void> {
  // Verify webhook signature first
  if (!this.verifyWebhookSignature(req)) {
    return res.status(401).json({ error: 'Unauthorized' });
  }
```